project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/c4m_user_notifications.module

```diff
@@ -350,7 +350,14 @@ function c4m_user_notifications_flag_flag($flag, $entity_id, $account, $flagging
   }
 
   $node = node_load($entity_id);
-  $notification_types = array('wiki_page', 'discussion', 'document', 'event');
+  $notification_types = array(
+    'article',
```

Comment:
@soniCaH,
For notification for articles and news, need to make sure that users are subscribed to relevant flag.